src/libraries/Microsoft.Extensions.Options/tests/SourceGenerationTests/Baselines/NetCoreApp/Validators.g.cs

```diff
@@ -0,0 +1,1756 @@
+                    }
+                }
+            }
+
+            if (options.P4 != null)
+            {
+                var count = 0;
+                foreach (var o in options.P4)
+                {
+                    builder.AddResult(global::Enumeration.__ThirdModelValidator__.Validate(baseName + $"P4[{count++}]", o));
+                }
+            }
+
+            if (options.P5 != null)
+            {
+                var count = 0;
+                foreach (var o in options.P5)
+                {
+                    if (o is not null)
+                    {
+                        builder.AddResult(global::Enumeration.__ThirdModelValidator__.Validate(baseName + $"P5[{count++}]", o.Value));
+                    }
+                }
+            }
+
+            if (options.P51 != null)
+            {
+                var count = 0;
+                foreach (var o in options.P51)
+                {
+                    if (o is not null)
+                    {
+                        builder.AddResult(global::Enumeration.__ThirdModelValidator__.Validate(baseName + $"P51[{count++}]", o.Value));
+                    }
+                }
+            }
+
+            if (options.P6 != null)
+            {
+                var count = 0;
+                foreach (var o in options.P6.Value)
+                {
+                    if (o is not null)
+                    {
+                        builder.AddResult(global::Enumeration.__SecondModelValidator__.Validate(baseName + $"P6[{count++}]", o));
+                    }
+                    else
+                    {
+                        builder.AddError(baseName + $"P6[{count++}] is null");
+                    }
+                }
+            }
+
+            {
+                var count = 0;
+                foreach (var o in options.P7)
+                {
+                    if (o is not null)
+                    {
+                        builder.AddResult(global::Enumeration.__SecondModelValidator__.Validate(baseName + $"P7[{count++}]", o));
+                    }
+                    else
+                    {
+                        builder.AddError(baseName + $"P7[{count++}] is null");
+                    }
+                }
+            }
+
+            if (options.P8 != null)
+            {
+                var count = 0;
+                foreach (var o in options.P8.Value)
+                {
+                    if (o is not null)
+                    {
+                        builder.AddResult(global::Enumeration.__SecondModelValidator__.Validate(baseName + $"P8[{count++}]", o));
+                    }
+                    else
+                    {
+                        builder.AddError(baseName + $"P8[{count++}] is null");
+                    }
+                }
+            }
+
+            return builder.Build();
+        }
+    }
+}
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Do we still need this pragma or is that some R9 left overs?

Reply:
This was left over from the code I ported from R9. I am going to remove it as I don't think it is needed.